corehq/apps/api/resources/auth.py

```diff
@@ -62,7 +62,11 @@ def __init__(self, allow_session_auth=False, *args, **kwargs):
             }
 
     def is_authenticated(self, request, **kwargs):
-        return self._auth_test(request, wrappers=[self._get_auth_decorator(request), api_auth], **kwargs)
+        return self._auth_test(request, wrappers=[
+            self._get_auth_decorator(request),
```

Comment:
Does this need to be there twice? or does having it in the `require_permission` cover this?

Reply:
Did some more testing and it looks like you're right this is redundant. Removed it from the `require_permission` call in 278094946e1

Reply:
Reverted 2780949 because it broke several tests.

`corehq.apps.api.tests.internal_resources:InternalLocationResourceTest.test_basic` started seeing a 302 redirect instead of a 200, and `corehq.apps.api.tests.ucr_resources:TestSimpleReportConfigurationResource.test_auth` started getting a 404 instead of a 401. They're still a success and failure, respectively, but I'd rather not change behavior.